osmaxx-py/osmaxx/utilities/shortcuts.py

```diff
@@ -0,0 +1,34 @@
+from django.shortcuts import _get_queryset
+
+
+def get_object_or_none(klass, *args, **kwargs):
+    """
+    Uses get() to return an object, or None if the object
+    does not exist.
+
+    klass may be a Model, Manager, or QuerySet object. All other passed
```

Comment:
If you get an instance of a meta class, `klass` or `meta_class` is ok. 
If it is an instance of an arbitrary not yet known class, it should be named like `object`, `class_instance` or simple `instance`.